src/ElectricPokemon.java

```diff
@@ -0,0 +1,54 @@
+public class ElectricPokemon extends pokemonsuper{
+    private int electricCharge;
+    private boolean alive;
```

Comment:
Deze variabele heb je in elke subklasse geimplementeerd. JE had het daarom beter in de superklasse kunnen zetter en hier iets anders, unieks kunnen kiezen.